config.json

```
@@ -485,6 +485,19 @@
           "loops",
           "math"
         ]
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "ddbd717d-a6b5-40fc-9dc8-3526e7409091",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "status": "wip",
```

I think it's fine to merge this with `active`

```suggestion
        "status": "active",
```